src/counterfactuals/initialisation.jl

```diff
@@ -8,19 +8,7 @@ Initializes the starting point for the factual(s):
 """
 function initialize_state(ce::CounterfactualExplanation)
     @assert ce.initialization ∈ [:identity, :add_perturbation]
-
     s′ = ce.s′
-    data = ce.data
-
-    # No perturbation:
-    if ce.initialization == :identity
-        return s′
-    end
-
-    # If latent space, initial point is random anyway:
-    if ce.params[:latent_space]
-        return s′
-    end
```

Comment:
The function returns unmodified `s′` by default, so we don't need if-statements for the cases where we want to return `s′`, we only need if-statements for the cases where we want to return something else.

Reply:
Wait, now I noticed that `:latent_space` can be combined with the `:add_perturbations` initialization type. Since it doesn't matter for `:latent_space` whether perturbations are added or not, though, it still seems more consistent to always just do that if initialization is set to `:add_perturbations`.